packages/php-wasm/scopes/src/index.ts

```diff
@@ -12,6 +14,25 @@
  * For more information, see the README section on scopes.
  */
 
+/**
+ * Checks if the URL pathname is scoped.
+ *
+ * @example
+ * ```js
+ * isUrlPathnameScoped('/scope:my-site/index.php');
+ * // true
+ *
+ * isUrlPathnameScoped('/index.php');
+ * // false
+ * ```
+ *
+ * @param  url - The URL pathname to check.
+ * @returns `true` if the URL pathname is scoped, `false` otherwise.
+ */
+export function isUrlPathnameScoped(url: string): boolean {
+	return url.startsWith('/scope:');
```

Comment:
nit: If we expect this string to be a pathname, it might be good to use a more specific name than `url`. Maybe `pathname` or `uriPath` would be a good name.

```suggestion
 * @param  pathname - The URL pathname to check.
 * @returns `true` if the URL pathname is scoped, `false` otherwise.
 */
export function isUrlPathnameScoped(pathname: string): boolean {
	return pathname.startsWith('/scope:');
```